src/index.ts

```diff
@@ -59,6 +63,37 @@ declare global {
     }
 }
 
+//Add no-highlight class only when mouse is pressed on a div to ensure that elements in the div are
+//not highlighted any other time
+
+header?.addEventListener("mousedown", () => {
+    header.classList.remove("no-highlight");
+});
+header?.addEventListener("mouseleave", () => {
+    header.classList.add("no-highlight");
+});
+
+toolBar?.addEventListener("mousedown", () => {
+    toolBar.classList.remove("no-highlight");
+});
+toolBar?.addEventListener("mouseleave", () => {
+    toolBar.classList.add("no-highlight");
+});
+
+subBar?.addEventListener("mousedown", () => {
+    subBar.classList.remove("no-highlight");
+});
+subBar?.addEventListener("mouseleave", () => {
+    subBar.classList.add("no-highlight");
+});
+
+sideBar?.addEventListener("mousedown", () => {
+    sideBar.classList.remove("no-highlight");
+});
+sideBar?.addEventListener("mouseleave", () => {
+    sideBar.classList.add("no-highlight");
+});
```

Comment:
Id prefer to see this shortened with a helper function that takes the bar and event name, or just bar and adds both etc. .